neuroimg/pipeline/visualization/Snakefile

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+sys.path.append("../../../")
+from neuroimg.pipeline.fileutils import SCRIPTS_UTIL_DIR
+from neuroimg.pipeline.fileutils import get_freesurfer_patient_dir, get_freesurfer_dir, \
+    get_rawct_dir, get_rawmri_dir, get_rawacpc_dir
+configfile: "../config/localconfig.yaml"
+FS_OUTPUT_DIR = get_freesurfer_dir(config)
+FS_PATIENT_OUTPUT_DIR = get_freesurfer_patient_dir(config)
+
+rule aseg:
+	params:
+		subject = "derivatives/freesurfer/PublicDataset",
+		subjectName = "PublicDataset",
+		subjectDirectory = FS_OUTPUT_DIR
+	shell:
+		"./scripts/aseg2srf.sh -s {params.subject} && "
+		"./scripts/objMaker.sh {params.subject} && "
+		"mris_convert {params.subjectDirectory}/PublicDataset/surf/lh.pial {params.subjectDirectory}/PublicDataset/surf/lh.pial.asc && "
+		"mris_convert {params.subjectDirectory}/PublicDataset/surf/rh.pial {params.subjectDirectory}/PublicDataset/surf/rh.pial.asc && "
+		"cp {params.subjectDirectory}/{params.subjectName}/surf/lh.pial.asc {params.subjectDirectory}/{params.subjectName}/surf/lh.pial.srf && "
+		"cp {params.subjectDirectory}/{params.subjectName}/surf/rh.pial.asc {params.subjectDirectory}/{params.subjectName}/surf/rh.pial.srf && "
```

Comment:
Is the aseg2srf, the same file used to get out subcortical structures?

If so, we can asjust this to get what you need in a post-processing step.